packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -67,13 +67,16 @@ export class FASTAccordion extends FASTElement {
         this.$emit("change", this.activeid);
     };
 
-    private findExpandedItem(): FASTAccordionItem | null {
+    private findExpandedItem(): FASTAccordionItem | Element {
         for (let item: number = 0; item < this._accordionItems.length; item++) {
-            if (this._accordionItems[item].getAttribute("expanded") === "true") {
-                return (this._accordionItems[item] as unknown) as FASTAccordionItem;
+            if (
+                ((this._accordionItems[item] as unknown) as FASTAccordionItem)
+                    .expanded === true
+            ) {
+                return this._accordionItems[item];
             }
         }
-        return null;
+        return this._accordionItems[0];
```

Comment:
Will `this._accordionItems[0]` always exist in this situation?

Reply:
I moved the deafult down but this is a good catch. I think the whole file probably needs to guard against it better.

Reply:
I've added a couple guards to ensure we don't operate and will push up the changes.